### PR TITLE
[16.0][FIX] account_payment_partner: payment mode compute

### PR DESCRIPTION
**Pull request by ljsalvatierra-factorlibre**

### CODE REVIEW

**Review by aritzolea — APPROVED** [review]

File: account_payment_partner/models/account_move.py [hunk 1]
```diff
@@ -65,23 +65,35 @@ def _compute_payment_mode_id(self):
...
                     move.payment_mode_id = partner.supplier_payment_mode_id
-                elif move.move_type == "out_invoice":
+                elif (
+                    move.move_type == "out_invoice" and partner.customer_payment_mode_id
+                ):
                     move.payment_mode_id = partner.customer_payment_mode_id
                 elif (
                     move.move_type in ["out_refund", "in_refund"]
                     and move.reversed_entry_id
+                    and move.reversed_entry_id.payment_mode_id
```
Comment (pedrobaeza) [inline account_payment_partner/models/account_move.py]: No need of this line (and the same for the other 2 similar ones in following diffs)

Comment (ljsalvatierra-factorlibre) [reply account_payment_partner/models/account_move.py]: I'll take a look thank you.

Comment (ljsalvatierra-factorlibre) [reply account_payment_partner/models/account_move.py]: Done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**